distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFtp.java

```diff
@@ -264,4 +276,145 @@ private void showServerReply(FTPClient ftpClient) {
             }
         }
     }
-}
+
+    /**
+     * Client with keystore should be able to connect and successfully complete SSL handshake
+     * with FTP server when clientAuth = "want"
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testFtpsSslHandshakeWantClientAuthWithKeystore() throws Exception {
+        safeSetClientAuthConfiguration("want");
```

Comment:
Even though it's a test, it might be nice to define this with an enum or at least a static for "want" and "need"